core/api/export/replenishment.py

```diff
@@ -26,10 +26,18 @@ def write_data(self, data):
             record_row.append(self.write_record_cell(cell_c))
 
             self.sheet.append(record_row)
+        self.sheet.merge_cells("A4:C4")
```

Comment:
Please leave at least a comment stating why we need to do this. It's really not obvious. :)

Reply:
Sorry, you're right.